src/github.com/couchbase/sync_gateway/rest/changes_api.go

```diff
@@ -126,22 +141,31 @@ func (h *handler) handleChanges() error {
 	// The default is all channels the user can access.
 	userChannels := channels.SetOf(channels.AllChannelWildcard)
 	if filter != "" {
-		if filter != "sync_gateway/bychannel" {
-			return base.HTTPErrorf(http.StatusBadRequest, "Unknown filter; try sync_gateway/bychannel")
-		}
-		if channelsArray == nil {
-			return base.HTTPErrorf(http.StatusBadRequest, "Missing 'channels' filter parameter")
-		}
-		var err error
-		userChannels, err = channels.SetFromArray(channelsArray, channels.ExpandStar)
-		if err != nil {
-			return err
-		}
-		if len(userChannels) == 0 {
-			return base.HTTPErrorf(http.StatusBadRequest, "Empty channel list")
+		if filter == "sync_gateway/bychannel" {
+			if channelsArray == nil {
+				return base.HTTPErrorf(http.StatusBadRequest, "Missing 'channels' filter parameter")
+			}
+			var err error
+			userChannels, err = channels.SetFromArray(channelsArray, channels.ExpandStar)
+			if err != nil {
+				return err
+			}
+			if len(userChannels) == 0 {
+				return base.HTTPErrorf(http.StatusBadRequest, "Empty channel list")
+			}
+		} else if filter == "_doc_ids" {
```

Comment:
For consistency, I think we might want to make this "sync_gateway/_doc_ids", or even "sync_gateway/byDocId".  I don't think we need strict compatibility with the equivalent CouchDB filter.  What do you think?